florist/tests/integration/api/test_train.py

```diff
@@ -57,28 +59,35 @@ def test_train():
 
                 # Check response
                 assert response.status_code == 200
-                assert response.json() == {"server_uuid": ANY, "client_uuids": [ANY]}
+                assert response.json() == {
+                    "server_uuid": ANY, "client_uuids": [ANY]}
```

Comment:
Same here, breaking it into two lines instead of this reads better IMO.

Reply:
Oh god the whole file is formatted like this 😂 I have mixed feelings for autoformatting...

Reply:
Yeah sorry about that, I will fix it  and disable in future :)